product_brand/views/product_brand_view.xml

```diff
@@ -0,0 +1,194 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<odoo>
+    <record id="product_brand_search_form_view" model="ir.ui.view">
+        <field name="name">product.brand.search.form</field>
+        <field name="model">product.brand</field>
+        <field name="arch" type="xml">
+            <search string="Product Brand">
+                <field name="name"/>
+                <field name="partner_id"/>
+            </search>
+        </field>
+    </record>
+
+    <act_window
+        id="action_open_brand_products"
+        name="Brand Products"
+        res_model="product.template"
+        view_mode="kanban,form,tree"
+        domain="[('product_brand_id', '=', active_id)]"/>
+
+    <act_window
+        id="action_open_single_product_brand"
+        name="Product Brand"
+        res_model="product.brand"
+        view_mode="kanban,form,tree"
+        target="current"
+        domain="[('product_ids', 'in', active_id)]"/>
+
+    <record id="view_product_brand_form" model="ir.ui.view">
+        <field name="name">product.brand.form</field>
+        <field name="model">product.brand</field>
+        <field name="arch" type="xml">
+            <form>
+                <sheet>
+                    <div class="oe_button_box" name="button_box">
+                        <button name="%(action_open_brand_products)d"
+                            type="action"
+                            class="oe_stat_button"
+                            icon="fa-cubes">
+                            <field name="products_count" widget="statinfo" string="Products"/>
```

Comment:
If someone creates a product from here, the product should have the default_product_brand_id field on it already set to this brand.